config/cars/mods/urd/urd_t5_maures_2018.ini

```diff
@@ -0,0 +1,92 @@
+[INCLUDE]
+INCLUDE=common/no_popup_lights.ini
+
+[LIGHTING]
+INTERIOR_FAKE_UPPER_SHADOW_HEIGHT=0.03
+INTERIOR_FAKE_UPPER_SHADOW_FADE=0.05
+EMISSIVE_HEADLIGHTS_MULT=3
+EMISSIVE_PARKINGLIGHTS_MULT=2
+EMISSIVE_BRAKELIGHTS_MULT=4
+
+[BOUNCED_INTERIOR_LIGHT]
+MULT=1,1,1,1
```

Comment:
Why are you adding white bounced light to the interior?

Reply:
I assume the original author (URD) copied these values from one of the examples linked in the CSP wiki.